date.js

```diff
@@ -76,10 +79,19 @@ function getDayOfTheWeek(date) {
  * 
  */
 function getFormattedDate(date) {
+    let year = date.getFullYear().toString().substr(-2);
     let month = date.getMonth();
-    // etc...
-    // Your Code Here!
-    return `${month}/ etc...`;
+    let dayOfMonth = date.getDate();
+    let hour = date.getHours();
+    let minute = date.getMinutes();
+    let noon = 12;
+    let ampm = hour >= noon ? 'pm' : 'am' 
+    //how would you write this w/o a ternary operator??
+    hour = hour % noon;
+    if (hour === 0){
+        hour = 12;
```

Comment:
nit-picky, but for clean code-sake, maybe make a variable `let midnight = 12` at the top of the function, and here set `hour = midnight`. that way you don't have any "magic numbers"